arrow-libs/core/arrow-core/api/android/arrow-core.api

```diff
@@ -1199,13 +1233,19 @@ public final class arrow/core/raise/Trace {
 }
 
 public final class arrow/core/raise/context/RaiseContextualKt {
-	public static final fun accumulate (Larrow/core/raise/Raise;Lkotlin/jvm/functions/Function1;)Ljava/lang/Object;
-	public static final fun accumulating (Larrow/core/raise/RaiseAccumulate;Lkotlin/jvm/functions/Function1;)Larrow/core/raise/RaiseAccumulate$Value;
+	public static final fun accumulate (Larrow/core/raise/Accumulate;Ljava/lang/Object;)Larrow/core/raise/RaiseAccumulate$Value;
+	public static final fun accumulate (Larrow/core/raise/Raise;Lkotlin/jvm/functions/Function2;)Ljava/lang/Object;
+	public static final fun accumulateAll-EgRvm48 (Larrow/core/raise/Accumulate;Ljava/util/List;)Larrow/core/raise/RaiseAccumulate$Value;
+	public static final fun accumulating (Larrow/core/raise/Accumulate;Lkotlin/jvm/functions/Function2;)Larrow/core/raise/RaiseAccumulate$Value;
```

Comment:
I don't fully understand why this went from `Function1` to `Function2` in the API.